docs/lab/02/02.md

```diff
@@ -644,6 +644,64 @@ If the MCU provides timers, Embassy-rs will use them to suspend the software ins
 
 :::
 
+## Build and flash
+
+### Build
+
+To build a program for the Raspberry Pi Pico, run the following command (in the project root folder):
+
+`cargo build --bin <project_name>`
+
+where `<project_name>` is to be replaced with the name of your project.
+
+This will create the binary file inside `/target/thumbv6m-none-eabi/debug/`.
+
+### Flash
+
+To load your program to the Raspberry Pi Pico, connect the board through USB to your computer while holding down the `BOOTSEL` button. It should appear as an external drive on your system. Afterwards, you can run the following command (also from the root folder):
+
+`elf2uf2-rs -d .\target\thumbv6m-none-eabi\debug\<project_name>`
```

Comment:
```suggestion
```shell
elf2uf2-rs -d target/thumbv6m-none-eabi/debug/<crate_name>
```
```

The / delimiter works for both windows and linux systems.

Reply:
Changed all paths.